src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -180,6 +180,7 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, scratchPool, layerThickness, laye
       !$omp barrier
 
       nCells = nCellsArray(2)
+      !$omp do schedule(runtime) private(i, iEdge)
```

Comment:
@mark-petersen - I'm not sure I understand the omp master section above, why some allocates are inside it and others are not